src/near-rpc-provider.ts

```diff
@@ -372,4 +373,17 @@ export class NearRpcProvider extends JsonRpcProvider {
       })
     }
   }
+
+  async getValidatorStatus(blockHashOrBlockNumber: string[] | number[] | null[]): Promise<GetValidatorStatusResponse> {
```

Comment:
I think maybe a better API for the end user is to provide a string or a number (or nothing) and we can then wrap those values in an array ourselves. 
So the type for this method would be `async getValidatorStatus(blockHashOrBlockNumber?: string | number)`
And then when when doing the `send`, we can wrap the block in an array, something like: `send('validators', [blockHashOrBlockNumber || null])`

This way the user can use it by doing `provider.getValidatorStatus('my-validator')`

Reply:
Ohh, nice! I'll fix this right now!